pkg/repository/v1/trigger.go

```diff
@@ -103,11 +103,25 @@ func newTriggerRepository(s *sharedRepository) TriggerRepository {
 	}
 }
 
-func (r *TriggerRepositoryImpl) TriggerFromEvents(ctx context.Context, tenantId string, opts []EventTriggerOpts) ([]*sqlcv1.V1Task, []*DAGWithData, error) {
+type EventDataForOlap struct {
+	EventKey                string
+	EventId                 string
+	TaskExternalId          string
```

Comment:
If we decide to keep this, `TaskExternalId` isn't as accurate since it could be a DAG external id as well

Reply:
good call, will remove / replace this